packages/jsii/lib/runtime-info.ts

```diff
@@ -0,0 +1,100 @@
+import * as spec from '@jsii/spec';
+import * as ts from 'typescript';
+
+/**
+ * A limited subset of the Map<ts.ClassDeclaration, spec.TypeBase> to ease testing.
+ */
+export interface RuntimeClassInfo {
+  get(key: ts.ClassDeclaration): spec.TypeBase | undefined;
```

Comment:
Feels a little unnecessary to take complete `spec.TypeBase` objects if you only need the FQN.